src/en/users-auth.md

```diff
@@ -0,0 +1,45 @@
+Title: Juju users and authentication
+TODO: Stuff on user-created models (ssh key and credentials)
+
+
+# Users and authentication
+
+This section deals with topics related to user authentication and general user
+security.
+
+
+## Users and passwords
+
+There are two ways users are introduced into Juju: the initial administrator of
+a controller via the controller creation step and a non-administrator via the
+user registration step. The latter sets up the user's Juju password but the
+former is left without an actual password.  This is why if such a user tries to
+log out (`juju logout`) before creating a password the command will fail and a
+warning will be displayed. An administrator should, therefore, create a
+password once the controller is created. They must do so if multiple Juju users
+will be using the same system user account:
+
+```bash
+juju bootstrap mycontroller aws
+juju change-user-password
+```
+
+See [Creating a controller](./controllers-creating.html) for details on
+controller creation.
+
+
+## Credentials and SSH keys
+
+*Credentials* managed in Juju are related to the accessing of the chosen
+cloud-backed resource. They are not related to Juju users themselves in any
+way. Credentials added to Juju remain local to each user (on Ubuntu:
```

Comment:
possible confusion here. Credentials remain local to the user of the Juju client. 'user' is very ambiguous here